src/sidebar/components/menu-item.js

```diff
@@ -56,57 +57,74 @@ export default function MenuItem({
   const leftIcon = isSubmenuItem ? null : renderedIcon;
   const rightIcon = isSubmenuItem ? renderedIcon : null;
 
-  return (
-    <Fragment>
-      {/* FIXME-A11Y */}
-      {/* eslint-disable-next-line jsx-a11y/role-supports-aria-props */}
-      <div
-        aria-checked={isSelected}
-        className={classnames('menu-item', {
+  let menuItem = null;
+
+  if (href) {
+    // link item
+    //
+    menuItem = (
+      <a
+        className={classnames('menu-item__action', {
           'menu-item--submenu': isSubmenuItem,
           'is-disabled': isDisabled,
           'is-expanded': isExpanded,
           'is-selected': isSelected,
         })}
-        role="menuitem"
+        href={href}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        {hasLeftIcon && (
+          <div className="menu-item__icon-container">{leftIcon}</div>
+        )}
+        <span className={labelClass}>{label}</span>
+        {hasRightIcon && (
+          <div className="menu-item__icon-container">{rightIcon}</div>
+        )}
+      </a>
+    );
+  } else {
+    // button item
+    //
+    menuItem = (
+      // eslint-disable-next-line jsx-a11y/role-supports-aria-props
+      <button
```

Comment:
not using the Button component here because we need to have some complex inner children.